### ISSUE TITLE
make class parameters consistent with GradientBoostingClassifier

### ISSUE THREAD

**Issue by asmahani** [asmahani/ordinal-boost#10]
Examples:

- max_iter -> n_estimators
- lr_g -> learning_rate (or learning_rate_g)
- reltol (should we change this to 'tol'? perhaps not since our definition is most likely different from scikit-learn)

potential new features:

- subsample
- loss (function)
